orka-utils.c

```diff
@@ -130,6 +130,24 @@ orka_iso8601_to_unix_ms(char *timestamp, size_t len, void *p_data)
   return 1; // SUCCESS
 }
 
+int
+orka_unix_ms_to_iso8601(char *str, size_t len, void *p_data)
+{
+  uint64_t *p_timestamp = (uint64_t*)p_data;
```

Comment:
Really good. But to spare an extra line you can rename p_data to p_timestamp and throw that in the ASSERT_S